krun/tests/test_scheduler.py

```diff
@@ -35,24 +38,43 @@ def dummy_do_reboot(self):
         from logging import info
         info("SIMULATED: reboot via exception")
         raise TestReboot()
-    monkeypatch.setattr(ExecutionScheduler, '_do_reboot', dummy_do_reboot)
+    monkeypatch.setattr(util, '_do_reboot', dummy_do_reboot)
 
 
+def no_envlogs(monkeypatch):
+    def dummy_stash_envlog(tmp_filename, config, platform, key, exec_num):
+        pass
+    monkeypatch.setattr(util, 'stash_envlog', dummy_stash_envlog)
+
```

Comment:
Extraneous blank line? [It's a bit unclear to me if the standard in this file is 1 or 2 blank lines between functions.]